WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -0,0 +1,74 @@
+import Gridicons
+
+@objc class CreateButtonCoordinator: NSObject {
+
+    private enum Constants {
+        static let padding: CGFloat = -16
+        static let heightWidth: CGFloat = 56
+    }
+
+    var button: FloatingActionButton = {
+        let button = FloatingActionButton(image: Gridicon.iconOfType(.create))
+        button.accessibilityLabel = NSLocalizedString("Create", comment: "Accessibility label for create floating action button")
+        button.accessibilityIdentifier = "floatingCreateButton"
+        button.accessibilityHint = NSLocalizedString("Creates new post or page", comment: " Accessibility hint for create floating action button")
+        return button
+    }()
+
+    private weak var viewController: UIViewController?
+
+    let newPost: () -> Void
+    let newPage: () -> Void
+
+    @objc init(_ viewController: UIViewController, newPost: @escaping () -> Void, newPage: @escaping () -> Void) {
+        self.viewController = viewController
+        self.newPost = newPost
+        self.newPage = newPage
+    }
+
+    /// Should be called any time the `viewController`'s trait collections change
+    /// - Parameter previousTraitCollect: The previous trait collection
+    @objc func presentingTraitCollectionDidChange(_ previousTraitCollection: UITraitCollection) {
+    }
```

Comment:
Do we still need this method, since it's empty now?

Reply:
It will be used in the follow up PR and I didn't want to remove it from both places. Trying to cut down a bit on line changes. I've got about 700-800 once the action sheet UI + custom presentation code is added (that API is so verbose! 😣).

Reply:
Thanks for adding the TODO!